.github/workflows/reg.yml

```diff
@@ -1,15 +1,21 @@
-name: Visual Regression Testing
+name: Visual Regression Testing and preview environment
 
-on: [push]
+on:
+  push:
+    branches:
+      - 'master'
+      - 'beta'
+  pull_request: {}
```

Comment:
can't leave a comment if its not on 'pull request' but also the old conf was building regsuit on every push in every branch